app/assets/javascripts/utils.js

```diff
@@ -1,3 +1,34 @@
+(function () {
+  function initInstitutionCollectionDropdowns() {
```

Comment:
Wondering whether we should move this to a JS file specific to the reports page.

Reply:
"I was trying to make this function generic so it could be re-used in other places as well. That's the reason I have put it in utils" Since institutions and collections are linked I can see how this might work. Ok, let's try it out.